frontend/src/App.tsx

```diff
@@ -15,7 +17,8 @@ export default function App() {
     <StyledMain>
       <Routes>
         <Route path="/" element={<Homepage />}/>
-        <Route path="/members" element={<MembersOverview />}/>
+        <Route path="/add/members" element={<AddMemberForm />}/>
+        <Route path="/list/members" element={<MemberPage />}/>
```

Comment:
Im Frontend gibt es zwar bei den Pfaden nicht so klare Richtlinien wie mit REST im Backend, aber nichts desto trotz würde ich eine Struktur bevorzugen, bei der allgemeineres vorn und konkreteres hinten im Pfad steht.

```suggestion
        <Route path="/members/add" element={<AddMemberForm />}/>
        <Route path="/members/list" element={<MemberPage />}/>
```